contrib/docker/docker-entrypoint.sh

```diff
@@ -4,14 +4,18 @@ set -ex
 if [ "$#" -ne 0 ]; then
     exec "$@"
 elif [ "$SUPERSET_ENV" = "development" ]; then
-    superset worker &
+    celery worker --app=superset.sql_lab:celery_app --pool=gevent -Ofair &
```

Comment:
@mistercrunch can you validate it is the correct command? I have no idea :)

Reply:
command looks good